src/components/blocks/faq-with-categories.tsx

```
@@ -0,0 +1,118 @@
+"use client";
```

ESLint

[error] Replace `"use·client"` with `'use·client'`

(prettier/prettier)